egui/src/widgets/plot.rs

```diff
@@ -239,18 +287,19 @@ pub struct Plot {
     show_y: bool,
 }
 
-impl Default for Plot {
-    fn default() -> Self {
+impl Plot {
+    pub fn new(name: impl Into<String>) -> Self {
```

Comment:
I think this should be either changed to `id_source: impl Hash`, or we should actually use it as a `String` - i.e. show it on hover. Though then I wonder what the difference would be between a plot name and `on_hover_ui`-tooltip

Reply:
How about we use the same system that `Window` uses with a constructor that takes the shown name and an `id` method to set a custom id when required? The title will be visible to the user at some point, I just haven't thought about how to show it yet.

Reply:
How about we use the same system that Window uses with a constructor that takes the shown name and
an id method to set a custom id when required? The title will be visible to the user at some point,
I just haven't thought about how to show it yet.